pkg/chains/formats/intotoite6v2/intotoite6v2.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package intotoite6v2
```

Comment:
Probably makes more sense to have this be reflected in the directory structure - `intotoite6/v2` (or `slsa/v1` if we end up going with that)